libs/dh/admin/feature-user-management/src/lib/tabs/users-tab/dh-users-tab-status-filter.component.ts

```diff
@@ -59,28 +65,16 @@ export class DhUsersTabStatusFilterComponent implements OnInit {
 
   @Output() changed = new EventEmitter<MarketParticipantUserStatus[]>();
 
-  userStatusOptions: WattDropdownOption[] = [];
+  userStatusOptions: WattDropdownOption[] = Object.keys(MarketParticipantUserStatus).map(
+    (entry) => ({
+      value: entry,
+      displayValue: entry,
+    })
+  );
 
   ngOnInit(): void {
-    this.buildUserStatusOptions();
     this.statusControl.valueChanges
       .pipe(takeUntilDestroyed(this._destroyRef))
```

Comment:
Suggestion: I think it is possible to move this to a constructor, then it would be possible to remove the destoryRef